enhancements/mcd-reboot-exceptions.md

```diff
@@ -0,0 +1,247 @@
+---
+title: mcd-reboot-dictionary
```

Comment:
The word "dictionary" does not appear anywhere else in this document. It is unclear what the name "mcd-reboot-dictionary" means.

Reply:
Oh, that was the old title. Will look for any other occurrences too